man/subscription-manager.8

```diff
@@ -580,19 +580,19 @@ option
 .B --remove=NAME
 Removes a named override from the repositories specified with the
 .B --repo
-option
+option. The override will not be removed if its source is one of the environments the system belongs in. It can only be removed by either: removing the override from the environment it originates from or removing the system from the environment the override originates from.
 
 .TP
 .B --remove-all
-Removes all overrides from repositories specified with the
+Removes a named override from the repositories specified with the
```

Comment:
There is a copy/paste artifact in this line. It should be `Removes all overrides from repositories..` instead of `Removes a named override from the repositories..`